About.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles1.css">
+    <title>Mobile World Congress 2023</title>
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <a href=""><i class="fa fa-facebook"></i></a>
+            <a href=""><i class="fa fa-twitter"></i></a>
+            <a href="">French</a>
+            <a href="">My Page</a>
+            <a href="">Barcelona</a>
+        </div>
+        <img src="./img/hamburger.svg" class="menu" alt="menu">
+
+        <nav class ="Desktop-navbar">
+             
+            <a href="./index.html"><img src="./img/MWClogo.png" alt="MWC" class="mylogo" /></a>
+            
+            <ul class="desktop-menu">
+
+                <li class="about"><a class="about" href="./About.html">About</a></li>
+                <li class=""><a href="">Program</a></li>
+                <li class=""><a href="">Join</a></li>
+                <li class=""><a href="">Sponsor</a></li>
+                <li class=""><a href="">News</a></li>
+                <a href=""><button type="button">CC Campaign</button></a>
```

Comment:
- [OPTIONAL] On the _**About**_ Page of the application, it would be nice to remove the link to the _**About**_ page and replace it with the link to the _**Home**_ page. I noticed you had a link to the _**Home**_ page for the `mobile menu` but on the `desktop` version, it shows the link to the _**About**_ page. 😎

Reply:
Hi @Gambit142 I will replace it, thank you  😎

Reply:
@Hisoka37, you are welcome 👍